terraform/modules/tf_threat_intel_downloader/dynamodb.tf

```diff
@@ -26,89 +26,12 @@ resource "aws_dynamodb_table" "threat_intel_ioc" {
   }
 }
 
-// IAM Role: Application autoscalling role
-resource "aws_iam_role" "stream_alert_dynamodb_appautoscaling" {
-  count              = "${var.autoscale ? 1 : 0}"
-  name               = "${var.prefix}_streamalert_dynamodb_appautoscaling"
-  assume_role_policy = "${data.aws_iam_policy_document.appautoscaling_assume_role_policy.json}"
-}
-
-// IAM Policy Doc: Generic Application Autoscaling AssumeRole
-data "aws_iam_policy_document" "appautoscaling_assume_role_policy" {
-  count = "${var.autoscale ? 1 : 0}"
-
-  statement {
-    effect  = "Allow"
-    actions = ["sts:AssumeRole"]
-
-    principals {
-      type        = "Service"
-      identifiers = ["application-autoscaling.amazonaws.com"]
-    }
-  }
-}
-
-// IAM Role Policy: Allow appautoscaling IAM role to autoscaling DynamoDB table
-resource "aws_iam_role_policy" "appautoscaling_update_table" {
-  count  = "${var.autoscale ? 1 : 0}"
-  name   = "DynamoDBAppAutoscaleUpdateTablePolicy"
-  role   = "${aws_iam_role.stream_alert_dynamodb_appautoscaling.id}"
-  policy = "${data.aws_iam_policy_document.appautoscaling_update_table.json}"
-}
-
-// IAM Policy Doc: Allow autoscaling IAM role to send alarm to CloudWatch
-// and change table settings for autoscaling.
-// This policy is allow the role to change table settings
-data "aws_iam_policy_document" "appautoscaling_update_table" {
-  count = "${var.autoscale ? 1 : 0}"
-
-  statement {
-    effect = "Allow"
-
-    actions = [
-      "dynamodb:DescribeTable",
-      "dynamodb:UpdateTable",
-    ]
-
-    resources = [
-      "${aws_dynamodb_table.threat_intel_ioc.arn}",
-    ]
-  }
-}
-
-// IAM Role Policy: Allow appautoscaling IAM role to autoscaling DynamoDB table
-resource "aws_iam_role_policy" "appautoscaling_cloudwatch_alarms" {
-  count  = "${var.autoscale ? 1 : 0}"
-  name   = "DynamoDBAppAutoscaleCloudWatchAlarmsPolicy"
-  role   = "${aws_iam_role.stream_alert_dynamodb_appautoscaling.id}"
-  policy = "${data.aws_iam_policy_document.appautoscaling_cloudwatch_alarms.json}"
-}
-
-// IAM Policy Doc: This policy is allow the role to send alarm to CloudWatch.
-data "aws_iam_policy_document" "appautoscaling_cloudwatch_alarms" {
-  count = "${var.autoscale ? 1 : 0}"
-
-  statement {
-    effect = "Allow"
-
-    actions = [
-      "cloudwatch:PutMetricAlarm",
-      "cloudwatch:DescribeAlarms",
-      "cloudwatch:GetMetricStatistics",
-      "cloudwatch:SetAlarmState",
-      "cloudwatch:DeleteAlarms",
-    ]
-
-    resources = ["*"]
-  }
-}
-
 resource "aws_appautoscaling_target" "dynamodb_table_read_target" {
   count              = "${var.autoscale ? 1 : 0}"
```

Comment:
Can you also help remove this line and `autoscale` variable? Because deleting custom static IAM role, it implies the autoscaling is enable all the time and can not be disabled.

Reply:
There is another line below should be deleted.

Reply:
Good idea, thanks!